package.json

```diff
@@ -128,7 +128,7 @@
     "ws": "^8.11.0"
   },
   "engines": {
-    "node": ">=18.17"
+    "node": ">=20.18.1"
```

Comment:
what was the reason for this particular version?

Reply:
Because it's the latest version that was available at the time of the release. You should not be using _anything else_.

Reply:
> The latest Android version is 15, so the all developers should do applications only for Android 15?

This is not a good analogy. A good analogy is slightly different: should Google recommend the use of _anything_ but the latest patch available for that given version? As we speak, this is 15.0.0_r5. Why would Google recommend the use of a version that contains known vulnerabilities? You can find the details related to Android CVEs here: https://source.android.com/docs/security/bulletin/2024-11-01#spl-vulnerability-details.

Reply:
[`engines.node`](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#engines) field is not about "recommendations". 

The purpose of this field is to show which versions of Node.js the library will work on and which will not.

Wrong use of this field feels like not a recommendation, but a force to update Node.js.